yage-glutin/src/lib.rs

```diff
@@ -0,0 +1,8 @@
+pub mod application;
+pub use application::*;
+
+pub mod context;
+pub use context::*;
```

Comment:
Meta: I'm a bit worried about the name collisions. People will assume that `yage::glutin` is a re-export of glutin and that `yage::glutin::Context/Application` are actually the glutin structs.
On that note, we should actually re-export `glutin`. It's best practice (unless you want to strictly hide the dependency), because otherwise users have to manually match their glutin version to ours or they may experience inscrutable compile errors.

Reply:
Then we should rethink this. We also already have an inconsistency here, because when importing between the libraries, the module names need to be different. That's not a big deal, but together with this issues, we should rethink the whole thing.